conductor/src/conductor/models/cluster.py

```diff
@@ -19,3 +19,7 @@ class ClusterDB(DBModel, TimestampMixin, SurrogatePKUUID):
     cluster_type = Column(String(), nullable=False)
     status = Column(String(), nullable=False)
     cluster_caps = Column(String(), nullable=True)
+    # suggestion
+    messenger = Column(String(), nullable=False, default="GCP")
+    # for GCP this would be the topic
+    messenger_queue = Column(String(), nullable=False)
```

Comment:
I don't think we need this for now. It is true that `conductor` is agnostic to messenger implementations. But, once it is started, it is going to use a single implementation of `messenger`. Are you thinking of multiple messenger implementation being used by `conductor` simultaneously?

Reply:
We need to know where each cluster will publish the messages, we can either save that to the DB or follow a naming convention for the queues/topics. 
Example, create a pub / sub topic named after the cluster name, for example.

Reply:
I am thinking of using a naming convention.

Reply:
I think it is more flexible to store that on the DB, you can create test channels or debug channels and swap it on the DB for debugging a cluster for example.
Another option could be to have a config option somewhere to customize the names if needed. That would make that channel names more visible IMO.
:thinking:

Reply:
alright! Saving queue names to DB makes more sense. But, I am still not sure about the `messenger` field. Do you have any specific use cases in mind?

Reply:
Cool. We have GCP now, but that could change in the future. I can see how we can use rabbitmq or AWS SQS instead of GCP Pub/Sub. I was trying to get ahead of that. 

For example, suppose in the future EBI gets an AWS account and money to spend. We wouldn't switch this overnight,  we could switch one of the clusters to test it. Or even between GCP Pub/Sub versions, they could deprecate one client and give us one month to migrate, we could then switch cluster by cluster.